package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-jest-formatting",
-  "version": "0.0.11",
+  "version": "0.0.12",
```

Comment:
This should be a minor version bump, but either way I'd rather do the version bumping in a separate followup PR.

Reply:
Yep. Good catch